actionview/Rakefile

```diff
@@ -25,6 +26,32 @@ namespace :test do
     t.ruby_opts = ["--dev"] if defined?(JRUBY_VERSION)
   end
 
+  task :ujs do
+    begin
+      Dir.mkdir("log")
+      pid = spawn("bundle exec rackup test/ujs/config.ru -p 4567 -s puma > log/test.log 2>&1")
```

Comment:
Not something to worry about right now, but I wonder if this could be simplified (particularly, wrt the connectivity check below) if it just started rack in-line on a thread.